javascript/javascript3/week1/happy-project/script.js

```diff
@@ -0,0 +1,69 @@
+window.addEventListener("DOMContentLoaded", () => {
+    const alerts = document.querySelectorAll('.alert');
+
+    alerts.forEach((alert) => {
+        alert.style.display = "none";
+    })
+})
+
+const inputTagName = document.getElementById("inputName");
+const txtAreaMsg = document.getElementById("msg");
+
+const formTag = document.getElementById("my-form");
+
+const happy = document.getElementById("alert-happy");
+const sad = document.getElementById("alert-sad");
+const neutral = document.getElementById("alert-neutral");
+
+const userNames = document.querySelectorAll(".user-name");
```

Comment:
`document.getElementsByClassName` would also work here, FYI.

Reply:
> `document.getElementsByClassName` would also work here, FYI.

If I use `document.querySelectorAll`  it returns a static NodeList that can be looped over using directly `forEach` method later in the code. 

While `document.getElementsByClassName` returns a live HTMLCollection, not an array. And I would need to convert it to an array first.